galette/templates/default/elements/footer.html.twig

```diff
@@ -48,23 +32,23 @@
 {% else %}
                         <i class="icon cookie bite" aria-hidden="true"></i>
 {% endif %}
-                        Galette {{ constant('GALETTE_DISPLAY_VERSION') }}
+                        Galette{% if login.isLogged() or callstatic('\\Galette\\Core\\Galette', 'isDemo') %} {{ constant('GALETTE_DISPLAY_VERSION') }}{% endif %}
```

Comment:
I guess we should keep only displayed version when user is logged in; I try to keep demo up-to-date (espacially when there are security issues, of course), but I may fail ^^
```suggestion
                        Galette{% if login.isLogged() %} {{ constant('GALETTE_DISPLAY_VERSION') }}{% endif %}
```